spec/models/your_details_form_spec.rb

```diff
@@ -5,15 +5,7 @@
   let(:salary_frequency) { 'year' }
   let(:contribution_preference) { 'full' }
 
-  describe '.load_config' do
-    context 'file exists' do
-      let(:salary_threshold_config_file) do
-        Wpcc::Engine.root.join('config', 'salary_threshold.yml')
-      end
-
-      specify { expect(salary_threshold_config_file).to exist }
-    end
-
+  describe 'load configuration' do
```

Comment:
I don't think these tests are enough. You may want to test that at least 2 or 3 of the currently config files can be loaded successfully. You may always want to consider the cases where the file does not exist!

Reply:
Why will you try to load a file that don't exist?